src/utils/generateMoviesUrl.ts

```diff
@@ -0,0 +1,13 @@
+import { KEY } from '../constants/MOVIES_URL';
+
+const generateMoviesUrl = (baseUrl: string, params?: object) => {
+  const searchParamsURL = new URLSearchParams({
+    api_key: KEY as string,
+    language: 'ko-KR',
+    ...params,
+  });
+
+  return baseUrl + searchParamsURL;
```

Comment:
요 부분에 템플릿 리터럴을 활용할 수 있지 않을까요?
```suggestion
  return `${baseUrl}${searchParamsURL}`;
```